src/bot/plugins/week_routine.py

```diff
@@ -0,0 +1,66 @@
+import re
+
+import structlog
+from dependency_injector.wiring import Provide, inject
+from mmpy_bot import Message, Plugin, listen_to
+
+from src.bot.schemas import Action, Actions, Attachment, Integration
+from src.bot.services.week_routine import NotifyService
+from src.depends import Container
+
+FRIDAY_TIME_SENDING_MESSAGE = 11
+DAY_OF_WEEK_FRI = "fri"
+LOGGER = structlog.get_logger()
+
+
+class WeekRoutine(Plugin):
+    action_yes = Actions(
+        id="Yes", name="Да", type="botton", integration=Integration(url="", context=Action(action="yes"))
+    )
+
+    action_no = Actions(
+        id="No", name="Нет", type="botton", integration=Integration(url="", context=Action(action="no"))
+    )
+
+    every_friday_message = Attachment(
+        text="Хочешь ли принять участие в random coffee на следующей неделе?", actions=[action_yes, action_no]
+    )
+
+    @listen_to("/send_asking_message", re.IGNORECASE)
+    @inject
+    async def send_asking_message(
+        self, message: Message, routine_service: NotifyService = Provide[Container.week_routine_service]
```

Comment:
Теперь это `notify_service`
```suggestion
        self, message: Message, notify_service: NotifyService = Provide[Container.week_routine_service]
```